### PR TITLE
Feature/NAS-9 Show only one pool in the borrow page

### PR DESCRIPTION
**Pull request by LothricFin**

### CODE REVIEW

File: src/components/MetamaskChecker/MetamaskChecker.vue [hunk 1]
```diff
@@ -33,7 +33,9 @@ export default {
   methods: {
     async checkProvider() {
       const provider = await detectEthereumProvider();
-
+      console.log('--------------------');
+      console.log('provider', provider);
+      console.log('--------------------');
```
Comment (rnkq) [inline src/components/MetamaskChecker/MetamaskChecker.vue:38]: remove console logs

File: src/components/UiComponents/NetworkButton.vue [hunk 2]
```diff
@@ -36,11 +37,22 @@ export default {
...
+          chainid: "0x539",
+          title: "AVAX local",
+          icon: avaxIcon,
+        },
       ],
     };
   },
   computed: {
     activeNetwork() {
+      console.log('this.networkType', this.networkType);
```
Comment (rnkq) [inline src/components/UiComponents/NetworkButton.vue:55]: remove console log

**Review by rnkq — CHANGES_REQUESTED** [review]

File: src/components/MetamaskChecker/MetamaskChecker.vue (hunk 1 above)
Comment (LothricFin) [reply src/components/MetamaskChecker/MetamaskChecker.vue]: Done

File: src/mixins/pools.js [hunk 3]
```diff
@@ -121,9 +121,12 @@ export default {
...
       let userBalance;
       try {
-        userBalance = await tokenContract.balanceOf(this.account, {
-          gasLimit: 600000,
-        });
+        // userBalance = await tokenContract.balanceOf(this.account, {
+        //   gasLimit: 600000,
+        // });
+        userBalance = await this.$ethers
+          .getDefaultProvider("http://localhost:8545")
```
Comment (rnkq) [inline src/mixins/pools.js:128]: get provider from the state, do not hardcode here.

**Review by rnkq — CHANGES_REQUESTED** [review]

File: src/mixins/pools.js (hunk 3 above)
Comment (rnkq) [reply src/mixins/pools.js:128]: fixed in PR#4

**Pull request merged**